Documentation/ApiOverview/Fluid/Index.rst

```diff
@@ -109,23 +132,75 @@ you must define the sections that are referenced by the layout.
 :file:`Layouts`
 ---------------
 
+*optional*
+
 Layouts serve as a wrapper for a web page or a specific block of content. If using Fluid
 for a sitepackage, a single layout file will often contain multiple components such as your
 sites menu, footer, and any other items that are reused throughout your website.
 
+Templates can be used with or without a Layout.
+
+* *With a Layout* anything that's not inside a section is ignored. When a Layout is used,
+  the Layout determines which sections will be rendered from the template through use of
```

Comment:
the Layout determines which sections will be rendered from the template through the use of